envs/environment_py38_iris30.yml

```diff
@@ -21,11 +21,7 @@ dependencies:
   - scipy=1.6
   - sigtools
   - sphinx
-  - statsmodels
```

Comment:
Hi @tjtg. Can we add a comment to this environment stating what is NOT supported, so that we know what to expect. e.g. 
```
# This environment does not support the following CLIs:
- estimate-emos-coefficients
- generate-timezone-mask-ancillary
- nowcast-accumulate
- nowcast-extrapolate
- nowcast-optical-flow-from-winds
- generate-advection-velocities-from-winds
```